### ISSUE TITLE
Thanks for your shares

### ISSUE THREAD

**Issue by mehmetyukcell** [teewhydope/HACKINTOSH_HP_PROBOOK_430_440_450_G6_OPENCORE_EFI#1]
Thanks for your system folders, but, I want to İnstall big sur public beta with opencore? Possible? I used Hp probook 450 g6 i7 8575u.

**Comment by teewhydope** [comment]
create a usb installer here https://github.com/corpnewt/gibMacOS and install it the same way you installed catalina

**Comment by mehmetyukcell** [comment]
> create a usb installer here https://github.com/corpnewt/gibMacOS and install it the same way you installed catalina

Thanks... possible for Big Sur?

**Comment by teewhydope** [comment]
> > create a usb installer here https://github.com/corpnewt/gibMacOS and install it the same way you installed catalina
> 
> Thanks... possible for Big Sur?

yes, it will work. i've got 2 G6 probook users successfully upgraded to Big Sur. just make sure all the kexts are updated and you're using the latest opencore version

**Comment by mehmetyukcell** [comment]
I have Hp probook 450 g6 i7 8565u. I used Catalina 10.15.6 with Clover. I want İnstall big sur and ı have usb big sur public beta.

**Comment by teewhydope** [comment]
> I have Hp probook 450 g6 i7 8565u. I used Catalina 10.15.6 with Clover. I want İnstall big sur and ı have usb big sur public beta.

port to opencore first, if everything works fine on catalina, then you can think of upgrading. big sur works better on opencore

**Comment by teewhydope** [comment]
You can use also use this script to download full installer app from swscan.apple.com. quite easy

[installScript_110fullapp.zip](https://github.com/teewhydope/HACKINTOSH_HP_PROBOOK_440_G6_OPENCORE_EFI/files/5145360/installScript_110fullapp.zip)

**Comment by mehmetyukcell** [comment]
> You can use also use this script to download full installer app from swscan.apple.com. quite easy
> 
> [installScript_110fullapp.zip](https://github.com/teewhydope/HACKINTOSH_HP_PROBOOK_440_G6_OPENCORE_EFI/files/5145360/installScript_110fullapp.zip)

what should i do after i do this

**Comment by teewhydope** [comment]
after using the script to download, install the beta.app normally. remove all vodoo kexts except vodoops2controller, to avoid kernel panic

**Comment by teewhydope** [comment]
> > You can use also use this script to download full installer app from swscan.apple.com. quite easy
> > [installScript_110fullapp.zip](https://github.com/teewhydope/HACKINTOSH_HP_PROBOOK_440_G6_OPENCORE_EFI/files/5145360/installScript_110fullapp.zip)
> 
> what should i do after i do this

check here for full installation guide https://github.com/teewhydope/HACKINTOSH_HP_440_G6_EFI_OPENCORE_BIG_SUR
